mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -158,6 +168,138 @@ impl Nexus {
         }
     }
 
+    pub async fn start_rebuild(
+        &mut self,
+        destination: &str,
+    ) -> Result<(), Error> {
+        trace!("{}: start rebuild request for {}", self.name, destination);
+
+        let source = match self
```

Comment:
I see this pattern in several places, perhaps its an idea to create a function for this? Something like child_lookup()?